### ISSUE TITLE
missing run.py 

### ISSUE THREAD

**Issue by Nikarko01** [omkarcloud/google-maps-reviews-scraper#4]
### Description

missing run.py in root

### Steps to Reproduce

1. ls run.py

**Expected behavior:** [What you expect to happen]

finding run.py in root

**Actual behavior:** [What actually happens]

no run.py in root

**Reproduces how often:** [What percentage of the time does it reproduce?]

everytime

**Comment by easyspace** [comment]
Yup, same thing here. Even requirements.txt is missing. What a waste of time.

**Comment by Chetan11-dev** [comment]
Sorry, I forgot to update the project README last year. It's now updated, so please take a look at it.  
If you have any questions or would like the pro version free of cost, feel free to reach at chetan@omkar.cloud.
